### PR TITLE
add scheduled message process task

### PR DESCRIPTION
**Pull request by mdayican**
JIRA link (if applicable)
https://tools.hmcts.net/jira/browse/BPS-765

Change description

scheduled processor added
 
Does this PR introduce a breaking change? (check one with "x")

[ ] Yes
[x] No

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/bulkscan/payment/processor/service/task/PaymentMessageProcessTask.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
+package uk.gov.hmcts.reform.bulkscan.payment.processor.service.task;
```
Comment (doncem) [inline src/main/java/uk/gov/hmcts/reform/bulkscan/payment/processor/service/task/PaymentMessageProcessTask.java]: which packaging convention we choose here? i'm fine but perhaps better to lift it up?

File: src/main/java/uk/gov/hmcts/reform/bulkscan/payment/processor/service/task/PaymentMessageProcessTask.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+@Service
+@ConditionalOnProperty(value = "scheduling.task.consume-payments-queue.enabled", matchIfMissing = true)
+public class PaymentMessageProcessTask {
+
+    private static final Logger log = LoggerFactory.getLogger(PaymentMessageProcessTask.class);
+
+    private final PaymentMessageProcessor paymentMessageProcessor;
+
+    public PaymentMessageProcessTask(
+        PaymentMessageProcessor paymentMessageProcessor) {
```
Comment (doncem) [inline src/main/java/uk/gov/hmcts/reform/bulkscan/payment/processor/service/task/PaymentMessageProcessTask.java]: `)` should be in new line

File: src/integrationTest/resources/application-integration.yaml [hunk 3]
```diff
@@ -10,3 +10,8 @@ idam:
     url: *wiremock
   api:
     url: *wiremock
+
+scheduling:
+  task:
+    consume-payments-queue:
+      enabled: false
```
Comment (nitinprabhu) [inline src/integrationTest/resources/application-integration.yaml]: should we set it true ?

Comment (mdayican) [reply src/integrationTest/resources/application-integration.yaml]: we do not test in integration

File: src/test/java/uk/gov/hmcts/reform/bulkscan/payment/processor/service/task/PaymentMessageProcessTaskTest.java [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+    private PaymentMessageProcessor paymentMessageProcessor;
+
+    private PaymentMessageProcessTask queueConsumeTask;
+
+    @BeforeEach
+    public void setUp() {
+        queueConsumeTask = new PaymentMessageProcessTask(
+            paymentMessageProcessor
+        );
+
```
[the remark this answers is not in the document]
Comment (mdayican) [reply src/test/java/uk/gov/hmcts/reform/bulkscan/payment/processor/service/task/PaymentMessageProcessTaskTest.java]: removed

**Comment by mdayican** [comment]
will create new PR,this  one stuck in pipeline

**Pull request closed without merging**